kubetest/README.md

```diff
@@ -53,6 +53,9 @@ whatever changes you have made into a quick release.
 Control the details of the `--build=bazel` by appending one of the build modes
 (see help for current list).
 
+If `PWD` is in the `kubernetes/test-infra/kubetest` directory, build the binary
```

Comment:
this Build section is for how to build kubernetes - add it to #installation section above probably - (and `go get -u k8s.io/test-infra/kubetest` *should* work)

Reply:
`go get -u k8s.io/test-infra/kubetest` is working - i am adding an alternative here.